tap_postgres/sync_strategies/incremental.py

```diff
@@ -130,7 +131,15 @@ def _get_select_sql(params):
     table_name = params['table_name']
 
     limit_statement = f'LIMIT {params["limit"]}' if params["limit"] else ''
-    where_statement = f"WHERE {replication_key} >= '{replication_key_value}'::{replication_key_sql_datatype}" \
+
+    if not params["offset"] or not replication_key_value:
+        offset = ''
+    elif replication_key_sql_datatype.startswith('timestamp'):
+        offset = f' - interval \'{params["offset"]} seconds\''
+    else:
+        offset = f' - {params["offset"]}'
```

Comment:
I'm not really clear on this 

>It can be used by integer ID based replication as well, in that case it will import the last offset number of rows before the latest replicated id.

do you have an example? meaning if we do incrementing serial primary key, we put a value like "1000" and if the last ID is 9999 we will only replicate up to ID 8999?

Reply:
If the above is correct, I think maybe the term `offset` can be avoided, because it's very similar to pagination terminologies (limit and offset) yet behaves differently, especially the interval one.

There are also 2 units here (interval uses `seconds`, and Incrementing ID uses `count`) and they are both just expressed as number (`offset: N`). This could be confusing, maybe we can separate to two different config?

```
skip_last_n_rows: 1000
skip_last_n_seconds: 900
```

Reply:
sounds good! 👍